_examples/real-world-examples/synchronizing-databases/README.md

```diff
@@ -0,0 +1,63 @@
+# Synchronizing Databases (MySQL to PostgreSQL)
+
+This example shows how to use [SQL Pub/Sub](https://github.com/ThreeDotsLabs/watermill-sql) across two different databases.
+
+See also [SQL Pub/Sub documentation](https://watermill.io/pubsubs/sql).
+
+## Background
+
+Synchronizing two databases can be a tough task, especially with different data formats.
+This example shows how to migrate a MySQL table to PostgreSQL table using watermill.
+
+The application will first transfer all existing rows and then keep listening for any new inserts,
+copying them to the new table as soon, as they appear.
```

Comment:
it may be good to mention explicitly, that updates and deletes are not supported